### ISSUE TITLE
Doesn't work fine with long modules and `\`

### ISSUE THREAD

**Issue by brickgao** [timothycrosley/isort#273]
There is an example for long modules:

``` python
from testtesttest import iamaveryveryveryvertveryveryveryvertveryveryveryverylong1, iamaveryveryveryvertveryveryveryvertveryveryveryverylong2
```

After `isort`:

``` python
from testtesttest import (iamaveryveryveryvertveryveryveryvertveryveryveryverylong1,
    iamaveryveryveryvertveryveryveryvertveryveryveryverylong2)
```

Maybe sort to these will be better:

``` python
from testtesttest import \
    (iamaveryveryveryvertveryveryveryvertveryveryveryverylong1,
     iamaveryveryveryvertveryveryveryvertveryveryveryverylong2)
```

And `isort` not work fine with `. After I sorted above code, it turned to

``` python
from testtesttest import \
    iamaveryveryveryvertveryveryveryvertveryveryveryverylong1

     iamaveryveryveryvertveryveryveryvertveryveryveryverylong2)
```

**Comment by timothycrosley** [comment]
@brickgao,

Thanks for reporting this. I agree that the suggested output would be a good format for super long lines, will try to add support for this before the next release.

Thanks!

~Timothy

**Comment by allanlewis** [comment]
I'm also seeing this with grid mode 2. Is there any progress to report on resolving this issue?

**Comment by jpmelos** [comment]
+1, I'm also seeing this issue.

**Comment by timothycrosley** [comment]
This should be fixed in the development version of isort, to be released as isort 5.0.0
